apps/src/applab/dataBrowser/AddTableRow.jsx

```diff
@@ -0,0 +1,33 @@
+import Radium from 'radium';
+import React from 'react';
+import * as dataStyles from './dataStyles';
+
+const AddTableRow = React.createClass({
+  propTypes: {
+    columnNames: React.PropTypes.array.isRequired
+  },
+
+  render() {
+    return (
+      <tr style={dataStyles.addRow}>
+        {
+          this.props.columnNames.map(columnName => (
+            <td key={columnName} style={dataStyles.cell}>
+              { (columnName !== 'id') && <input style={dataStyles.input}/> }
+            </td>
+          ))
+        }
+        <td style={dataStyles.cell}>
+          <button
+            className="btn btn-primary"
+            style={dataStyles.button}
```

Comment:
I haven't been able to make RubyMine do this for me, but it looks like we might be able to catch this in precommit hooks via https://github.com/yannickcr/eslint-plugin-react